packages/vite-plugin-svelte/src/index.js

```diff
@@ -185,9 +189,43 @@ export function svelte(inlineOptions) {
 			async buildEnd() {
 				await options.stats?.finishAll();
 			}
-		},
-		svelteInspector()
+		}
 	];
+	if (!isSvelte5) {
+		plugins.push(svelteInspector()); // TODO reenable once svelte5 has support
+	}
+	if (isSvelte5) {
+		log.warn(
+			'svelte 5 support in vite-plugin-svelte is experimental, breaking changes can occur in any release until this notice is removed'
+		);
+		log.warn('svelte5 does not support svelte-inspector yet, disabling it');
```

Comment:
```suggestion
		log.warn('svelte 5 does not support svelte-inspector yet, disabling it');
```

Reply:
refactored so it's a single log and changed wording a bit